person.rb

```diff
@@ -0,0 +1,18 @@
+# Person class
+class Person
+  def initialize(age, name = 'Unknown', _parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+  end
+
+  attr_reader :id, :name, :age
+
+  def can_use_services?
+    @parent_permission || is_of_age?
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] It would be better if our method `is_of_age` is a private method. So that it can not be accessed by any of the classes